src/runtime/components/data/Table.vue

```diff
@@ -109,6 +111,9 @@ export default defineComponent({
       type: Object as PropType<{ column: string, direction: 'asc' | 'desc' }>,
       default: () => ({})
     },
+    fnSort: {
+      type: [ Function]
+    },
```

Comment:
What you think about adding `fnSort` to main `srot` prop like this 

```js
    sort: {
      type: Object as PropType<{ column: string, direction: 'asc' | 'desc', sortBy?: Function }>,
      default: () => ({})
    },
```

```js

if (props.sort?.sortBy) {
  sort.value.column = toSort.column
  await props.sort.sortBy(toSort)
}

sort.value = toSort
```

Reply:
Better 
```
     <UTable
        :sort="{
          column: sortData.column,
          direction: sortData.direction,
          sortBy: fnSort
        }"
        :loading="loading"
        :columns="columns"
        :rows="data"
      >
        <template #url-data="{ row }">
          <UAvatar :src="row.url" alt="Avatar" />
        </template>
      </UTable>
```

Reply:
Yeah, it looks more readable to avoid a lot of props in features